src/Tests/Microsoft.NET.Build.Tests/AppHostTests.cs

```diff
@@ -16,17 +17,45 @@
 using Microsoft.NET.TestFramework.ProjectConstruction;
 using Xunit;
 using Xunit.Abstractions;
+using NuGet.Frameworks;
 
 namespace Microsoft.NET.Build.Tests
 {
     public class AppHostTests : SdkTest
     {
+        private static string[] GetExpectedFilesFromBuild(TestAsset testAsset, string targetFramework)
+        {
+            var testProjectName = testAsset.TestProject?.Name ?? testAsset.Name;
+            var expectedFiles = new List<string>()
+            {
+                $"{testProjectName}{Constants.ExeSuffix}",
+                $"{testProjectName}.dll",
+                $"{testProjectName}.pdb",
+                $"{testProjectName}.deps.json",
+                $"{testProjectName}.runtimeconfig.json"
+            };
+
+            if (!string.IsNullOrEmpty(targetFramework))
+            {
+                var parsedTargetFramework = NuGetFramework.Parse(targetFramework);
+                if (parsedTargetFramework.Version >= new Version(5, 0, 0, 0))
+                    expectedFiles.Add($"ref/{testProjectName}.dll");
+
+                if (parsedTargetFramework.Version < new Version(6, 0, 0, 0))
+                    expectedFiles.Add($"{testProjectName}.runtimeconfig.dev.json");
+            }
+
+            return expectedFiles.ToArray();
+        }
+
         public AppHostTests(ITestOutputHelper log) : base(log)
         {
         }
 
-        [PlatformSpecificTheory(TestPlatforms.Windows | TestPlatforms.Linux | TestPlatforms.FreeBSD)]
-        [InlineData("netcoreapp3.0")]
+        [PlatformSpecificTheory(TestPlatforms.Windows | TestPlatforms.Linux | TestPlatforms.FreeBSD | TestPlatforms.OSX)]
```

Comment:
Now that this runs on macOS, can this just be a `Theory` instead of a `PlatformSpecificTheory`?